### DISCUSSION TITLE
Unable to get Student Pack

### DISCUSSION THREAD

**Discussion by joe-joeh** [community/community#51756] [GitHub Education]
Error: You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
My attempts: tried enormous amount of different bios, full name was always included in both full name and bio, i have no clue what is required.

**Comment by subeom7** [comment]
Did you use your school's email to verify? I was able to resolve this issue by using my school's email. 

Also, your student ID should display your expected graduation date. If it does not, ensure that you verify your enrollment using alternative school documents that confirm your graduation date

Additionally, for the "How do you plan to use GitHub?" section, make sure to say you are open to contributing to open-source projects.

Lastly, link your institution's webpage: In the "Company" or "Website" section of your profile, consider adding a link to your educational institution's webpage to further validate your academic affiliation.


After making these updates, reapply for the GitHub Student Pack. If you continue to face difficulties, consider reaching out to GitHub Support directly, as they can provide more specific guidance or investigate any issues with your application:

GitHub Education Support: https://support.github.com/contact/education

**Reply by joe-joeh** [reply]
True, was using edu mail and the document has expected graduation date. Probably the issue is that uploaded photo is my studentship ticket, which is written in Ukrainian, though i have my friend who uploaded just same document but was easily approved

**Reply by LizaKoch** [reply]
how did you solved this problem?
